packages/core/http/core-http-router-server-internal/src/versioned_router/mocks.ts

```diff
@@ -19,5 +19,5 @@ export function createRouter(): jest.Mocked<IRouter> {
     patch: jest.fn(),
     routerPath: '',
     versioned: {} as any,
-  };
+  } as unknown as Router;
```

Comment:
NIT: `as unknown as jest.Mocked<Router>` ? (but not sure it's necessary given it seems we don't use the mock capabilities it seems)

Reply:
5a35aa9cd48